linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/HttpInitializerTest.scala

```diff
@@ -154,6 +156,7 @@ class HttpInitializerTest extends FunSuite with Awaits with Eventually {
     assert(sparams[hparam.MaxInitialLineSize] == maxInitLineSize)
     assert(sparams[hparam.MaxRequestSize] == maxReqSize)
     assert(sparams[hparam.MaxResponseSize] == maxRspSize)
+    assert(sparams[Headers.param.MaxErrResponseSize] == maxErrRspSize)
```

Comment:
`Headers.param.MaxErrResponseSize -> param.MaxErrResponseSize` would probably look a bit better

Reply:
renamed to hErrParam